lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/LessonsAccess.java

```diff
@@ -68,11 +70,23 @@
     static final boolean useCache = true;
 
     // Sakai Service Beans
-    private SimplePageToolDao dao;
-    private AuthzGroupService authzGroupService;
-    private SecurityService securityService;
-    private MemoryService memoryService;
-    private UserDirectoryService userDirectoryService;
+    @Setter private SimplePageToolDao dao;
+    @Setter private MessageLocator messageLocator;
+    @Setter private AuthzGroupService authzGroupService;
+    @Setter private SecurityService securityService;
+    @Setter private MemoryService memoryService;
+    @Setter private SessionManager sessionManager; 
+    @Setter private ToolManager toolManager;
+    @Setter private UserDirectoryService userDirectoryService;
+    @Setter private UserTimeService userTimeService;
+    @Setter private SiteService siteService;
+    @Setter private ContentHostingService contentHostingService;
+    @Setter private GradebookIfc gradebookIfc;
+
+    @Setter private LessonEntity forumEntity;
+    @Setter private LessonEntity quizEntity;
+    @Setter private LessonEntity assignmentEntity;
+    @Setter private LessonEntity bltiEntity;
```

Comment:
I'd just stick @Setter on the class, tbh, but no big deal.

Reply:
I think I had it like this because originally some of the methods had getters so some of this was @Getter @Setter. But in reviewing this I realized that none of the getters were being used so just removed them. 

These changes weren't even relevant to the fix, I just don't like to look at useless get and set methods. 

I kind of feel like having it here isn't bad, and goes a little better with the "Explicit is better than implicit." Zen of Python rule. But I'm fine changing it too if annotating the class it the Sakai preferred style.